spec/models/comment_spec.rb

```diff
@@ -18,17 +18,29 @@
   end
 
   describe '#counts_for_requests' do
-    let(:request) { create :sequencing_request, asset: tube }
+    let(:request1) { create :sequencing_request, asset: tube }
     let(:tube) { create :multiplexed_library_tube }
 
-    before do
-      create :comment, commentable: tube, description: 'An excellent tube'
-      create :comment, commentable: tube.receptacle, description: 'A good receptacle'
-      create :comment, commentable: request, description: 'A reasonable request'
+    let(:request2) { create :sequencing_request }
```

Comment:
You don't need to create a request with a different name this way.  `context` blocks will override values just inside their block if you put the `let` inside there.  So you can leave line 21 called `request` and move this line (24) inside the `context` block starting on line 38 and also call it `request`.  The value will be overwritten, but only for the tests inside that `context`.

That's the whole point of `context`s, to set the fixture values up just for the tests inside its block.